testBinding.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: admissionregistration.k8s.io/v1alpha1
+kind: ValidatingAdmissionPolicyBinding
+metadata:
+  name: "demo-binding-test.example.com"
+spec:
+  policyName: "demo-policy.example.com"
+  matchResources:
+    namespaceSelector:
+      matchLabels:
+        environment: test
```

Comment:
Reminder to remove if used only for local testing. If used in test infrastructure, it can be moved to appropriate directories under `test/`

Reply:
Yeah I removed all these, had some issues with the integration tests so snuck those in quick but they're already removed

Reply:
Apologies! looks like I was reviewing from a very stale view of the PR.